admin/option_values.php

```diff
@@ -86,112 +108,87 @@
-            </tr>
-          </table>
-          <?php echo '</form>'; ?>
-          <?php
-        } // which table
-        ?>
+                echo zen_draw_form('update', FILENAME_PRODUCTS_OPTIONS_VALUES, 'action=update_sort_order&options_id=' . $_GET['options_id'], 'post', 'class="form-horizontal"'); ?>
+                <table class="table-condensed">
+                    <tr class="dataTableHeadingRow">
+                        <th class="dataTableHeadingContent text-center"><?= TABLE_HEADING_OPTION_VALUE_ID; ?></th>
+                        <th class="dataTableHeadingContent"><?= TABLE_HEADING_OPTION_VALUE_NAME; ?></th>
+                        <th class="dataTableHeadingContent text-center"><?= TEXT_SORT_ORDER; ?></th>
+                    </tr>
+                    <?php
+                    $rows = $db->Execute(
+                        'SELECT *
+                                  FROM ' . TABLE_PRODUCTS_OPTIONS_VALUES . ' pov,
+                                       ' . TABLE_PRODUCTS_OPTIONS_VALUES_TO_PRODUCTS_OPTIONS . ' povtpo
+                                  WHERE povtpo.products_options_values_id = pov.products_options_values_id
+                                  AND povtpo.products_options_id = ' . (int)$_GET['options_id'] . '
+                                  AND pov.language_id = ' . (int)$_SESSION['languages_id'] . '
+                                  ORDER BY pov.products_options_values_sort_order, pov.products_options_values_id'
+                    );
+                    foreach ($rows as $row) { ?>
+                        <tr>
+                            <td class="dataTableContent text-center"><?= $row['products_options_values_id']; ?></td>
+                            <td class="dataTableContent"><?= $row['products_options_values_name']; ?></td>
+                            <td class="dataTableContent"><?= zen_draw_input_field('options_values_new_sort_order[' . $row['products_options_values_id'] . ']', $row['products_options_values_sort_order'], 'size="4" class="form-control text-right"'); ?>
+                            </td>
+                        </tr>
+                        <?php
+                    } ?>
+                </table>
+                <div class="row">
+                    <?php
+                    if (count($languages_array) > 1) { ?>
+                        <div class="radio"><label><?= zen_draw_radio_field('language', 'current_only') . TEXT_UPDATE_SORT_LANGUAGE_CURRENT; ?></label></div>
+                        <div class="radio"><label><?= zen_draw_radio_field('language', 'all', true) . TEXT_UPDATE_SORT_LANGUAGE_ALL; ?></label></div> <?php
+                    } ?>
+                    <button type="submit" class="btn btn-warning"><?= TEXT_UPDATE_SUBMIT; ?></button>
+                    <a href="<?= zen_href_link(FILENAME_PRODUCTS_OPTIONS_VALUES); ?>" class="btn btn-default" role="button"><?= IMAGE_CANCEL; ?></a>
+                </div>
+                <?php echo '</form>'; ?>
```

Comment:
Since the rest of the changes are using short-echo (`<?='), this line should as well.

Reply:
My tendency was to use them only when embedded in html, where the improvement is clear.
I actually think the full tag is clearer when in isolation, but I'll go with the flow.